apps/designer/app/routes/$.tsx

```diff
@@ -76,7 +76,16 @@ export const loader: LoaderFunction = async ({
       throw json("Page not found", { status: 404 });
     }
 
-    return { ...canvasData, env, mode };
+    const params = { ...canvasData.params };
+
+    if (env.RESIZE_ORIGIN != null) {
+      params.propTransforms = params.propTransforms ?? {};
+      params.propTransforms.Image = {
+        resizeOrigin: env.RESIZE_ORIGIN,
+      };
+    }
```

Comment:
these are technically not props, these are transformer configs, we configure a middleware, then the middleware will receive component props and know what to do with it.

Reply:
I think this datastructure makes more sense to me, wdyt?

```
canvasData.propTransforms = [
  {
    component: 'Image',
    config:  [resizeOrigin: env.RESIZE_ORIGIN,}
  }
]
```

Reply:
but looking at his all makes me wonder if we could do what you suggested initial - flat config for all transformers, transformer takes what it needs. In the worst case we can make this here work later, right?

Reply:
Ok transforms now are in a standalone folder, and just transforms https://github.com/webstudio-is/webstudio-designer/blob/8539fbf2f41d4763ff308da662106ed843164c88/packages/react-sdk/src/prop-transforms/index.ts#L1

In the future, it can be an argument to sdk components.

All logic to create transforms etc are  moved here 

https://github.com/webstudio-is/webstudio-designer/blob/32b33b2d176f5687d8c2a28b42663dc5b5cdcee9/packages/react-sdk/src/user-props/prop-transforms.ts#L27

And now more generic.

https://github.com/webstudio-is/webstudio-designer/blob/32b33b2d176f5687d8c2a28b42663dc5b5cdcee9/packages/react-sdk/src/tree/root.ts#L36